app/views/candidate_interface/invites/decline.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: @fac_invite_decline_reason_form, url: candidate_interface_update_decline_reason_path(@invite), method: :patch) do |f| %>
```

Comment:
As a rule, a page needs a title. This particular page should have a title with the error prefix so that if there is an error on the page, the title is prefixed with 'Error:'

We should also have a back button. The should be able to see the the reasons, return to the previous page and switch their answer to 'accept'.